pallets/benchmarks/Cargo.toml

```diff
@@ -0,0 +1,35 @@
+[package]
+name = "pallet-benchmarks"
```

Comment:
Shouldn't we call it `trappist-runtime-benchmarks` ? I'm afraid that calling it `pallet-benchmarks` gives the wrong expectation that it is generic / re-usable as-is.

Reply:
totally makes sense to me